junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestInvocationContext.java

```diff
@@ -24,18 +25,18 @@
 
 	private final ParameterizedTestNameFormatter formatter;
 	private final ParameterizedTestMethodContext methodContext;
-	private final Object[] arguments;
+	private final List<Named<Object>> arguments;
 
 	ParameterizedTestInvocationContext(ParameterizedTestNameFormatter formatter,
-			ParameterizedTestMethodContext methodContext, Object[] arguments) {
+			ParameterizedTestMethodContext methodContext, List<Named<Object>> arguments) {
 		this.formatter = formatter;
 		this.methodContext = methodContext;
 		this.arguments = arguments;
 	}
 
 	@Override
 	public String getDisplayName(int invocationIndex) {
-		return this.formatter.format(invocationIndex, this.arguments);
+		return this.formatter.format(invocationIndex, this.arguments.toArray());
```

Comment:
Since we're now always passing `Named` objects to the formatter, I think we should change the signature of the `format` method to reflect that.

Reply:
Tbh, I left `Object` on purpose because this method is used a lot in `ParameterizedTestNameFormatterTests`, and we'd have to wrap everything with `Named.of(...)` if we change the signature... and I found the result a bit heavy 😄 
But that's probably a wrong excuse, so I can do it anyway if you prefer 😁

Reply:
Well now that I think about it, if we follow your advice in https://github.com/junit-team/junit5/pull/2521#discussion_r567413100, we'll pass `Object[]` here anyway I think?